src/config/i18n/fr.json

```diff
@@ -0,0 +1,56 @@
+  "TITLE_AMBASSADOR": "Ambassadeur",
+
+  "HEADER_ABOUT": "À propos",
+  "HEADER_PARTNERS": "Nos solutions sont utilisées dans le monde entier",
+  "HEADER_TEAM": "L'Équipe",
+  "HEADER_FAQ": "Foire aux questions",
+  "HEADER_SUPPORTED_BY": "L'association Graasp est soutenue par",
+
+  "QUESTION_PLATFORMS": "Quelles sont les plates-formes de Graasp?",
+  "ANSWER_PLATFORMS": "La solution phare de l'Association Graasp est <0>Graasp</0>, une plate-forme d'éducation numérique ouverte accessible sur le Web, les appareils mobiles et les ordinateurs de bureau, intégrant une suite d'applications d'apprentissage.",
+  "QUESTION_GRAASP": "Qu’est-ce que Graasp?",
+  "ANSWER_GRAASP": "<0>Graasp</0> est une plate-forme sur laquelle vous pouvez créer vos propres activités ou ressources pédagogiques et numériques. Pour vous aider à démarrer, Graasp fournit des modèles de scénarios d'apprentissage que vous pouvez adapter à vos besoins. Vous pouvez soit partager ces ressources de manière privée avec vos élèves soit les mettre à disposition de quelques collègues ou de tous.",
+  "QUESTION_GETTING_STARTED": "Comment puis-je commencer?",
+  "ANSWER_GETTING_STARTED": "Vous pouvez essayer nos plates-formes gratuitement en vous inscrivant <0>ici</0>.",
+  "QUESTION_COST": "Combien ça coûte?",
+  "ANSWER_COST": "Vous pouvez commencer à utiliser <0>Graasp</0> gratuitement ! L'accès aux fonctionnalités premium, telles que les analytiques d'apprentissage et les espaces d'apprentissage personnalisés, coûte 5 CHF par mois, pour chaque utilisateur enregistré. Les institutions peuvent bénéficier de licences groupées ou d’hébergement sur site (en préparation) pour leurs enseignant-e-s. Si vous utilisez Graasp pour des activités à but non lucratif, vous pouvez bénéficier d’un soutien financier. Pour davantage d’informations, veuillez nous contacter à <1>contact@graasp.org</1>.",
+  "QUESTION_TRAINING": "Est-ce que vous proposez des formations?",
+  "ANSWER_TRAINING": "Oui ! Nous sommes conscients que les institutions et les enseignant-e-s ont des niveaux d'expérience variables en matière d'éducation numérique. Si vous souhaitez obtenir davantage d’informations sur nos formations personnalisées, veuillez nous contacter à <0>contact@graasp.org</0>.",
+  "QUESTION_BASE": "Où est basé Graasp?",
+  "ANSWER_BASE": "Nous sommes basés à Lausanne, en Suisse. Nos solutions sont également hébergées en Suisse et sont conformes à la réglementation européenne, y compris le RGPD.",
+
+  "NAV_ABOUT": "À propos",
+  "NAV_PARTNERS": "Partenaires",
+  "NAV_TEAM": "Équipe",
+  "NAV_FAQ": "FAQ",
+
+  "COOKIES_ACCEPT_BUTTON_TEXT": "J'accepte",
+  "COOKIES_TEXT": "Nous utilisons des cookies et d'autres technologies de suivi pour améliorer votre expérience de navigation sur notre site Web, pour analyser le trafic de notre site Web et pour comprendre d'où viennent nos visiteurs. En naviguant sur notre site Web, vous consentez à notre utilisation des cookies et autres technologies de suivi."
```

Comment:
These translations exist in common https://github.com/graasp/graasp-translations/blob/main/src/langs/fr/common.json 
I think you can delete yours and use the right key to refer to this other translations.

Reply:
OK! Yes, I will do that once I move these files to translations.